tests/validators/test_datetime.py

```diff
@@ -6,6 +6,8 @@
 
 import pytest
 import pytz
+from hypothesis import example, given
```

Comment:
Let's keep hypothesis tests in one place, move these tests to `test_hypothesis.py` and add conventional tests here.

Reply:
Can do. Given that I used hypothesis quite heavily here, how much overlap do you expect between the two? Perhaps it makes sense to only move `test_aware_and_naive_are_mutually_exclusive` and make the remaining ones test single values then?

Reply:
I don't think we need any hypothesis tests really.

In the end there are basically just 4 cases to test:
* require aware, input aware
* require aware, input naive
* require naive, input naive
* require naive, input aware

If you want to keep the hypothesis tests, please move them to `test_hypothesis.py`.

Reply:
if you are going to use hypothesis for these tests, it looks like you'll need to add `hypothesis[zoneinfo]` to deps.

TBH, I think it would probably be easier not to use hypothesis for these tests. I don't think it adds much here.